eclair-core/src/main/scala/fr/acinq/eclair/channel/Register.scala

```diff
@@ -79,6 +82,13 @@ class Register extends Actor with ActorLogging {
         case Some(channel) => channel.tell(msg, compatReplyTo)
         case None => compatReplyTo ! ForwardShortIdFailure(fwd)
       }
+
+    case GenerateLocalAlias(promise) =>
```

Comment:
Since we re-populate the `shortIds` map asynchronously as channels are restored and send `ShortChannelIdAssigned`, we may receive that message before we've restored all channels and recovered a complete `shortIds` map.

I believe it won't be an issue in practice as long as the range we use for `ShortChannelId.generateLocalAlias()` is big enough to make collisions unlikely.